homeassistant/components/vodafone_station/device_tracker.py

```diff
@@ -0,0 +1,133 @@
+"""Support for Vodafone Station routers."""
+from __future__ import annotations
+
+from collections.abc import Mapping
+from typing import Any
+
+from homeassistant.components.device_tracker import ScannerEntity, SourceType
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import _LOGGER, DEFAULT_DEVICE_NAME, DOMAIN
+from .coordinator import VodafoneStationDeviceInfo, VodafoneStationRouter
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up device tracker for Vodafone Station component."""
+
+    _LOGGER.debug("Start device trackers setup")
+    coordinator: VodafoneStationRouter = hass.data[DOMAIN][entry.entry_id]
+
+    tracked: set = set()
+
+    @callback
+    def update_router() -> None:
+        """Update the values of the router."""
+        add_entities(coordinator, async_add_entities, tracked)
+
+    entry.async_on_unload(
+        async_dispatcher_connect(hass, coordinator.signal_device_new, update_router)
+    )
+
+    update_router()
+
+
+@callback
+def add_entities(
```

Comment:
Please name this `async_`... (but it needs a new name)..maybe `async_add_new_tracked_entities`

Reply:
Here as well: https://github.com/home-assistant/core/blob/dev/homeassistant/components/asuswrt/device_tracker.py#L36